mgmt-hub/deploy-mgmt-hub.sh

```diff
@@ -447,37 +515,73 @@ if isMacOS; then
             fatal 2 "the commands jq and envsubst are required, and since brew is not installed, we can not install them for you"
         fi
     fi
-else   # ubuntu
-    echo "Updating apt package index..."
-    runCmdQuietly apt-get update -q
+else   # ubuntu and redhat
+    echo "Updating ${PKG_MNGR} package index..."
+    runCmdQuietly ${PKG_MNGR} update -q -y
     echo "Installing prerequisites, this could take a minute..."
-    runCmdQuietly apt-get install -yqf jq gettext-base make
+    runCmdQuietly ${PKG_MNGR} ${PKG_MNGR_INSTALL_QY_CMD} jq ${PKG_MNGR_GETTEXT} make
 
     # If docker isn't installed, do that
-    if ! command -v docker >/dev/null 2>&1; then
+    if ! isCmdInstalled docker; then
         echo "Docker is required, installing it..."
-        curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-        chk $? 'adding docker repository key'
-        add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
-        chk $? 'adding docker repository'
-        apt-get install -y docker-ce docker-ce-cli containerd.io
-        chk $? 'installing docker'
+        if isUbuntu18; then
+            curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+            chk $? 'adding docker repository key'
+            add-apt-repository "deb [arch=${ARCH_DEB}] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+            chk $? 'adding docker repository'
+            if [ "${ARCH}" = "amd64" ]; then
+                ${PKG_MNGR} install -y docker-ce docker-ce-cli containerd.io
+            elif [ "${ARCH}" = "ppc64le" ]; then
+                ${PKG_MNGR} install -y docker-ce containerd.io
+	        else
+	            fatal 1 "hardware plarform ${ARCH} is not supported yet"
+            fi
+            chk $? 'installing docker'
+        else # redhat
+            EPEL_VERSION=7              # The latest EPEL version for RHEL 7.9
+            DOCKER_VERSION=19.03.13     # The latest Docker version for RHEL 7.9 on pcc64le
```

Comment:
I'm still not comfortable with this. Just in the time this PR has been active, the docker version available has changed. How are we ever going to keep up with that? And how can a package repository not have a way to get the latest stable version of a package? That is central to every package repository.

The rest of this script defaults to pulling the latest version of packages and images, because the all-in-1 environment is meant for development, where you usually want to work with the latest code. I recognize that this might be a little more involved, so you can do this in a subsequent PR if you want (just let me know).

But at the very least, you need to add a variable near the top of the script, where the rest of the variables that can be overridden are listed:

```
export DOCKER_VERSION_PPC_RPM=${DOCKER_VERSION_PPC_RPM:-19.03.13}   # the latest available version can be found at ....
```

This accomplishes 2 things:
- Gets visibility to this near the top of this script where they tend to look for how to invoke/customize running this script
- Allows them to use a newer version w/o having to modify the script

Reply:
Fixed. Fortunately I have found a way to use the latest version in OpenPower repo. Close it.